### PR TITLE
UI::ServiceStatus: rename reload and enabled to xxx_flag

### PR DESCRIPTION
**Pull request by ancorgs**
To address the concerns expressed in https://github.com/yast/yast-yast2/pull/416

### CODE REVIEW

**Comment by ancorgs** [comment]
@mvidner @jreidinger It would be nice if you could take a quick look, so we submit this (and the related modules) before the next TW snapshot.

File: library/general/src/lib/ui/service_status.rb [hunk 1]
```diff
@@ -34,19 +34,19 @@ class ServiceStatus
     #   #name, #start, #stop, #enabled?, #running?
     #   For systemd compliant services, just do
     #   Yast::SystemdService.find("name_of_the_service")
-    # @param reload [Boolean] Initial value for the "reload" checkbox.
+    # @param reload_flag [Boolean] Initial value for the "reload" checkbox.
     #   Keep in mind it will always be displayed as unchecked if the service
     #   is not running, despite the real value.
-    # @param reload_label [Symbol] Type of label for the "reload" checkbox.
+    # @param reload_flag_label [Symbol] Type of label for the "reload" checkbox.
     #   :restart means that the service will be restarted, not just reloaded.
```
Comment (jreidinger) [inline library/general/src/lib/ui/service_status.rb]: document both possible values, so also `:reload` value in comment

**Comment by jreidinger** [comment]
LGTM

File: library/general/src/lib/ui/service_status.rb [hunk 2]
```diff
@@ -34,19 +34,20 @@ class ServiceStatus
     #   #name, #start, #stop, #enabled?, #running?
     #   For systemd compliant services, just do
     #   Yast::SystemdService.find("name_of_the_service")
```
Comment (mvidner) [inline library/general/src/lib/ui/service_status.rb]: > Note that this widget will #start and #stop the service by itself but the actions referenced by the checkboxes (reload and enable) are expected to be done by the caller, when the whole dialog is done.

**Comment by mvidner** [comment]
I suggest that one additional comment, otherwise LGTM.

**Pull request merged**